src/modules/RA/DataLayer/components/tabs/StyleTab/Style/WizardText.js

```diff
@@ -0,0 +1,199 @@
+import React from 'react';
+
+import { useTranslate, RadioButtonGroupInput } from 'react-admin';
+import { makeStyles } from '@material-ui/core/styles';
+import FormLabel from '@material-ui/core/FormLabel';
+
+import ColorStyleField from './ColorStyleField';
+import SizeStyleField from './SizeStyleField';
+import TextStyleField from './TextStyleField';
+
+import styles from './styles';
+
+const useStyles = makeStyles(styles);
+
+const fontList = [
```

Comment:
Naive question: in a perfect world, shouldn't it be in a config file or served by the back-end? Or is it something component related?